documentation/URL_SPEC.md

```diff
@@ -98,45 +98,19 @@ property pair being separated by **commas**.
-
-### Binning
-
-When loaded, each channel's raw intensity values are sorted into one of 256
-bins, where the bin index `0` holds the minimum raw intensity value of the
-channel and the bin index `255` holds the maximum raw intensity value. Like any
-binned histogram, each bin can represent multiple values-- if there are more
-than 256 intensities in the channel, values will be evenly distributed across
-bins.
-
-Certain properties like `cps` (control points), `rmp` (ramp), and `lut` (lookup
-table) directly reference these bin indices.
-
-![Screenshot of Volume Viewer with a volume in advanced mode. A histogram with
-255 bins is visible, with a ramp increasing from 0 to 1 on the Y axis in the
-span of 53 to 136 on the X axis.](./assets/example_histogram.png)
-
-For example, the above control points are at bin indices `0`, `53`, `136`, and
-`255`. Depending on the range of the channel's values, these may
-represent different raw intensity pixel values.
-
-| Min raw intensity | Max raw intensity | Bin `0` intensity | Bin `53` intensity | Bin `136` intensity | Bin `255` intensity |
-| ----------------- | ----------------- | ----------------- | ------------------ | ------------------- | ------------------- |
-| 0                 | 255               | 0                 | 53                 | 136                 | 255                 |
-| 50                | 200               | 50                | 103                | 166                 | 200                 |
-| 45                | 1000              | 45-48             | 244-247            | 555-558             | 997-1000            |
+| Property | Description                                                                       | Expected values                                                        | Default                                | Example                             |
+| -------- | --------------------------------------------------------------------------------- | ---------------------------------------------------------------------- | -------------------------------------- | ----------------------------------- |
+| `ven`    | Whether the volume is enabled.                                                    | `1` (enabled) or `0` (disabled)                                        | `0` (`1` for the first three channels) | `?c0=ven:1`                         |
+| `sen`    | Whether the isosurface is enabled.                                                | `1` (enabled) or `0` (disabled)                                        | `0`                                    | `?c0=iso:1`                         |
+| `isv`    | Isosurface value, the intensity value at which to generate the isosurface.        | Number in the range `[0, 255]`                                         | `128`                                  | `?c0=isv:195`                       |
```

Comment:
should `isv` change the range it names?  it is also a raw intensity now?  or is that going in a different pr?

Reply:
sounds like a breaking change...